client/src/LoginSignupComponents.js

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import { Button, TextField, InputLabel, Box } from "@material-ui/core";
+import { themes } from "./themes/LoginSignupThemes";
+
+function RedirectButton(props) {
+    const classes = themes();
+    return (
+        <Box className={props.className}
+            display="flex"
+            justifyContent="flex-end"
+            alignItems="baseline">
+            <span className={classes.redirectTextMargin}>{props.text}</span>
+            <Button className={classes.primary} href={props.href}>
+                {props.buttonText}
+            </Button>
+        </Box>
+    )
+}
```

Comment:
I don't think we need this level of granularity in making our components.

The entire sign up form can be something like:
```
<Grid container direction="column">
  <Grid>
    <Textfield /> // email stuff
  </Grid>
  <Grid>
    <Textfield /> // password stuff
  </Grid>
  <Grid>
    <Button /> // submit button
  </Grid>
</Grid>
```
Then this can be extended to use `FormControlLabel`, `InputLabel` etc. if you want.